packages/cli/src/commands/identity/withdraw-attestation-rewards.ts

```diff
@@ -19,7 +19,7 @@ export default class AttestationRewardsWithdraw extends BaseCommand {
   }
 
   async run() {
-    let {
+    const {
```

Comment:
hey @zviadm  it looks like this change is causing `yarn build` to break in `celocli`

Reply:
ah, lint complained about 'let', but didn't realize one of the variables is actually egtting changed.
i can't actually build this locally (it takes way to long to figure out which other subfolders i need to yarn build before i can build `cli`), but hopefully this next patch fixes it.